### ISSUE TITLE
Histograms (y-label and grouping)

### ISSUE THREAD

**Issue by milkha** [stan-dev/bayesplot#39]
1 - For histograms (e.g., in ppc_stat, and ppc_hist) it would be good to have a y-label which shows the counts (not sure what is plotted currently, counts or density). This is useful specially in ppc_stat_group, because it gives you an extra information about the size of each group.

2 - It would be good to have a ppc_hist_group function for comparing the distributions at a finer level.

**Comment by jgabry** [comment]
On Thursday, October 13, 2016, milkha <notifications@github.com> wrote:

> 1 - For histograms (e.g., in ppc_stat, and ppc_hist) it would be good to
> have a y-label which shows the counts (not sure what is plotted currently,
> counts or density). This is useful specially in ppc_stat_group, because it
> gives you an extra information about the size of each group.
>
Yeah, I'm not sure seeing the values is ever super useful (for this
particular plot) when it's density on the y-axis (that's what it is now),
but I agree it could be useful for counts. Maybe it should just always be
count, that way if you use `+ yaxis_text()` it will be meaningful for count
variables. Alternatively it could be an argument to the function, e.g.

ppc_stat_grouped(..., freq = TRUE),

which is how it is for `hist()` in base R.
>
> 2 - It would be good to have a ppc_hist_group function for comparing the
> distributions at a finer level.
>
Yeah I agree. Officially on the to-do list.

Thanks for the suggestions.

>

**Comment by jgabry** [comment]
@milkha I just changed the y-axis to counts in 54aaee2fac9415e6942f0d625b661c5cf459a5f2 for any PPC plot that is a histogram. The y-axis text is still off by default because it's really only useful in some cases, but it can easily be turned on using `yaxis_text()`, e.g. 
```r
ppc_stat(y, yrep, stat = "median") + yaxis_text()
```
should now show counts on the y-axis if you reinstall. 

I also moved `ppc_hist_grouped` to it's own issue (#41).
